azure-pipeline-templates/verbose-tests.yml

```diff
@@ -76,6 +80,23 @@ steps:
   - script: cat ${{ parameters.config }}
     displayName: Print config file
 
+# Stream e2e
```

Comment:
Should we also make a test case for read only config? Maybe one where we upload a bunch of files (with python SDK or something) then read all those files and validate their checksum or something?

Reply:
we can do that in a separate pr